lib/lambda_ethereum_consensus/fork_choice/helpers.ex

```diff
@@ -3,11 +3,29 @@ defmodule LambdaEthereumConsensus.ForkChoice.Helpers do
     Utility functions for the fork choice.
   """
   alias LambdaEthereumConsensus.StateTransition.{Accessors, Misc}
+  alias LambdaEthereumConsensus.Store.StateStore
   alias SszTypes.BeaconBlock
   alias SszTypes.BeaconState
   alias SszTypes.Checkpoint
   alias SszTypes.Store
 
+  @spec current_status_message(Store.t()) ::
+          {:ok, SszTypes.StatusMessage.t()} | {:error, any}
+  def current_status_message(store) do
+    with {:ok, head_root} <- get_head(store),
+         {:ok, state} <- StateStore.get_state(head_root) do
```

Comment:
you have `store.block_states`

Reply:
do you mean `StateStore.get_state` I believe there is no other function that i can fetch the head's state

Reply:
nvm Just found out I can take it trough here
```elixir
          blocks: %{SszTypes.root() => SszTypes.BeaconBlock.t()},
```